blinkstick/blinkstick.py

```diff
@@ -445,15 +450,23 @@ def set_led_data(self, channel, data):
         @param data: The LED data frame in GRB format
         """
 
-        report_id, max_leds = self._determine_report_id(len(data))
-
         report = [0, channel]
 
-        for i in range(0, max_leds * 3):
-            if len(data) > i:
-                report.append(data[i])
-            else:
-                report.append(0)
+        if self.mode == 3:
```

Comment:
Pack the data so each 24 bit RGB tuple is two interleaved 12 bit RGB tuples for mode 3